# This patch does the config changes specific to solaris.
# It is developed By solaris PSIF team. We plan to have a common upstream repo
# and submit these changes to it, but do not yet have a target date of doing it.
diff -r e1f52bdf4884 Makefile.am
--- a/Makefile.am	Thu Aug 25 00:19:37 2016 -0700
+++ b/Makefile.am	Thu Aug 25 00:26:27 2016 -0700
@@ -25,8 +25,6 @@
 sifconfdir = $(sysconfdir)/libibverbs.d
 sifconf_DATA = sif.driver
 
--include $(srcdir)/genrules.mk
-
 EXTRA_DIST = \
     libsif.spec.in \
     sif.driver \
